aspnetcore/grpc/interprocess.md

```diff
@@ -29,101 +35,18 @@ The preceding project file:
-    Host.CreateDefaultBuilder(args)
-        .ConfigureWebHostDefaults(webBuilder =>
-        {
-            webBuilder.UseStartup<Startup>();
-            webBuilder.ConfigureKestrel(options =>
-            {
-                if (File.Exists(SocketPath))
-                {
-                    File.Delete(SocketPath);
-                }
-                options.ListenUnixSocket(SocketPath, listenOptions =>
-                {
-                    listenOptions.Protocols = HttpProtocols.Http2;
-                });
-            });
-        });
-```
-
-The preceding example:
-
-* Configures Kestrel's endpoints in `ConfigureKestrel`.
-* Calls <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ListenUnixSocket%2A> to listen to a UDS with the specified path.
-* Creates a UDS endpoint that isn't configured to use HTTPS. For information about enabling HTTPS, see [Kestrel HTTPS endpoint configuration](xref:fundamentals/servers/kestrel/endpoints#listenoptionsusehttps).
-
-### Client configuration
-
-`GrpcChannel` supports making gRPC calls over custom transports. When a channel is created, it can be configured with a `SocketsHttpHandler` that has a custom `ConnectCallback`. The callback allows the client to make connections over custom transports and then send HTTP requests over that transport.
-
-Unix domain sockets connection factory example:
-
-```csharp
-public class UnixDomainSocketConnectionFactory
-{
-    private readonly EndPoint _endPoint;
-
-    public UnixDomainSocketConnectionFactory(EndPoint endPoint)
-    {
-        _endPoint = endPoint;
-    }
-
-    public async ValueTask<Stream> ConnectAsync(SocketsHttpConnectionContext _,
-        CancellationToken cancellationToken = default)
-    {
-        var socket = new Socket(AddressFamily.Unix, SocketType.Stream, ProtocolType.Unspecified);
-
-        try
-        {
-            await socket.ConnectAsync(_endPoint, cancellationToken).ConfigureAwait(false);
-            return new NetworkStream(socket, true);
-        }
-        catch
-        {
-            socket.Dispose();
-            throw;
-        }
-    }
-}
-```
-
-Using the custom connection factory to create a channel:
-
-```csharp
-public static readonly string SocketPath = Path.Combine(Path.GetTempPath(), "socket.tmp");
-
-public static GrpcChannel CreateChannel()
-{
-    var udsEndPoint = new UnixDomainSocketEndPoint(SocketPath);
-    var connectionFactory = new UnixDomainSocketConnectionFactory(udsEndPoint);
-    var socketsHttpHandler = new SocketsHttpHandler
-    {
-        ConnectCallback = connectionFactory.ConnectAsync
-    };
-
-    return GrpcChannel.ForAddress("http://localhost", new GrpcChannelOptions
-    {
-        HttpHandler = socketsHttpHandler
-    });
-}
-```
-
-Channels created using the preceding code send gRPC calls over Unix domain sockets. Support for other IPC technologies can be implemented using the extensibility in Kestrel and `SocketsHttpHandler`.
+## Inter-process communication transports
```

Comment:
Needed for the acronym in the third sentence.

I checked the live repo. There are no links to this section, so the heading change should be ok 🤞.

```suggestion
## Inter-process communication (IPC) transports
```